### PR TITLE
Added cache poisoning to known issues list, added language around beta product

### PR DESCRIPTION
**Pull request by H2RProductScience**
this is my first yourbase PR. Whee!

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -80,3 +80,17 @@ _Note that this since flag dramatically increases the tracing overhead, so we do
 ## Conflict with plugin pytest-xdist
 
 YourBase Test Acceleration and [pytest-xdist](https://pypi.org/project/pytest-xdist/) plugins have similar goals, reducing the overall test execution time of tests, but take different approaches to solving the problem. When they're both enabled, there are conflicts. When you're using YourBase Test Acceleration, please uninstall [pytest-xdist](https://pypi.org/project/pytest-xdist/) or execute pytest-xdist with `NUMCPUS=0`.
+
+---
+
+## Cache poisoning
+
+YourBase's acceleration cache can be poisoned when using command-line options to manually select tests (e.g. 'pytest path/to/test.py' or 'pytest -k name_of_test'). If a change causes another test to "want" to run but the test isn't selected by pytest, it will not be queued up to run next time it is.
```
Comment (dinamlev) [inline troubleshooting.md:88]: Would recommend using a backtick for the code snippets, e.g., 
`pytest path/to/test.py` or `pytest -k name_of_test`

**Review by dinamlev — APPROVED** [review]
I believe you are looking for review on this? Added myself as a reviewer.

File: troubleshooting.md (hunk 1 above)
Comment (H2RProductScience) [reply troubleshooting.md]: made this change

**Comment by H2RProductScience** [comment]
@glacials I added Dina's suggested changes and content around it being in Beta. Can you please review?

**Review by glacials — APPROVED** [review]
Looks good to me!

**Pull request merged**